pkgs/net-lib/info.rkt

```diff
@@ -8,8 +8,6 @@
 
 (define pkg-authors '(mflatt))
 
-(define version "1.1")
```

Comment:
I think this needs to be preserved, but it looks like the current version is 1.2, so we should bump it to 1.3 instead. Probably the build failure is because something else depends on it being at least 1.2.

Once this is bumped to 1.3, the history note in `server.scrbl` also needs to be updated to say `1.3`.

Reply:
Racket won't compile if you leave this in.

Reply:
the build fails because there is a duplicate version definition in the info.rkt that you had.

Reply:
Ohh, I see. Must've been a merge artifact. Still, we need to bump the version to 1.3 I think

Reply:
done